src/card.css

```diff
@@ -0,0 +1,84 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  width: 200px;
+  display: flex;
```

Comment:
delete all this styles
  display: flex;
  flex-direction: column;
  justify-content: center;
  font-style: normal;